### ISSUE TITLE
CTRL + Q and/or Gamepad Hotkeys for closing Cemu?

### ISSUE THREAD

**Issue by sonicnkt** [cemu-project/Cemu#592]
Just played around with the latest linux build (2.0-22) and it works pretty great on my system. 
I added a few games to steam so i can also launch them from my tv in another room using a gamepad.

This worked great as well but sadly i couldn't figure out a way to close the emulator without a keyboard/mouse as the steam overlay isn't working correctly

Some emulators support gamepad hotkeys (like yuzu for example) and for some other applications i use antimicrox  but even the last option didn't work for me. 

 (Almost) all desktop apps can be closed using CTRL + Q and with a tool like antrimicro i can easily add this as a button combo to a profile which is loaded before the emulator is started and removed afterwards. The mentioned key combo isn't working tho in the latest appimage.   

Adding Gamepad hotkeys is definitely a bigger implementation but maybe support for CTRL+Q (or another shortcut) could be added for working around this issue.

**Comment by jn64** [comment]
While I'm not against the suggestion, I've never noticed the lack of a close shortcut in any app, because practically every DE/WM has a global shortcut for closing windows. GNOME, KDE, and Xfce (xfwm) all have it as Alt-F4 by default (which I always change to Super-q).

Having "Quit Cemu" as a bindable option would be useful though, so you don't have to use an external tool to bind it to controller.

**Comment by sonicnkt** [comment]
Yeah on gnome it defaults to SUPER + Q which i am used to by now. Many Apps also support CTRL + Q tho. 
I am using antimicrox (flatpak), but the SUPER + Q combination isn't working here for some reason,  other button combinations (which don't use SUPER) work fine tho so this might be a gnome limitation. 
I discovered the CTRL-Q thing by accident and it worked in any other app (besides terminals). 

My solution for now is to keep using antimicrox with a more complicated setup:
Use GUIDE Button (Hold) to switch to another profile and here i mapped one button to ALT-ENTER and another button which switches to a third profile which enables mouse input so i can close the app manually. A bit complicated but nothing that can be accidentally triggered. 

Custom Hotkeys which support Gamepad input would be a great solution tho in the future.

**Comment by jn64** [comment]
For your workaround, surely you could do something simpler with xdotool or wmctrl to close the window?

```sh
# Filter by both classname and name to avoid closing the wrong window,
# like a browser window with "Cemu" in the title
xdotool search --classname 'Cemu' search --name 'Cemu 2.0' windowquit

# Or just quit the active window (may not be reliable, I don't know how antimicrox works)
xdotool getactivewindow windowquit
```

**Comment by Exzap** [comment]
This has been implemented in the upcoming 2.7 release
